services/src/handlers/ebv.rs

```diff
@@ -489,6 +507,23 @@ impl<C: Context> Task<C::TaskContext> for EvbMultiOverviewTask<C> {
         Ok(status.boxed())
     }
 
+    async fn cleanup_on_error(&self, _ctx: C::TaskContext) -> Result<(), Box<dyn ErrorSource>> {
+        // Tasks clean-up themselves on error.
+        // On abort, we propagate the abort to the current subtask.
+
+        let subtask_id = match self.current_subtask_id.lock().await.take() {
+            Some(id) => id,
+            None => return Ok(()), // no file was in progress, so nothing to clean up
+        };
+
+        // cannot be `force=true`, otherwise we wouldn't call this function
+        self.ctx
+            .tasks_ref()
+            .abort(subtask_id, false)
```

Comment:
so only the current subtask gets aborted and the other ones that are already completed remain? So the super task is not atomic?